packages/webapp/src/webview/ui/components/QuickFilters/QuickFilters.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import type { ReactElement } from 'react';
+import { VscFilter } from 'react-icons/vsc';
+import { useSelector } from 'react-redux';
+import type { AppState } from '../../../types';
+import { FocusZone, FocusZoneDirection } from '@fluentui/react-focus';
+import { actions } from '../../../state';
+import type { GuidedAnswersQueryFilterOptions } from '@sap/guided-answers-extension-types';
+import './QuickFilters.scss';
+
+/**
+ * Shows list of Quick Filters.
+ *
+ * @returns - react elements for QuickFilters view
+ */
+export function QuickFilters(): ReactElement {
+    const filters = useSelector<AppState, GuidedAnswersQueryFilterOptions[]>((state) => state.quickFilters);
+    return (
+        <div>
+            <h3 style={{ display: 'flex', alignItems: 'center' }}>
+                <VscFilter />
+                <span style={{ margin: '0 5px' }}>Quick Filters</span>
```

Comment:
@oliversberzs Can you please in one of your next PRs update here the text with i18n values? Thanks
https://github.com/SAP/guided-answers-extension/blob/main/packages/webapp/src/webview/i18n/i18n.json